maven/src/test/java/testfail/TestFail.java

```diff
@@ -1,10 +1,13 @@
 package testfail;
 
 import org.junit.Test;
+import org.junit.Ignore;
 
 import static org.junit.Assert.assertNotNull;
 
 public class TestFail {
+	
+	@Ignore //ignored so the entire tutorials build passes
```

Comment:
Hey @lor6 ,
This was supposed to fail but not fail the whole build as per this article https://www.baeldung.com/maven-ignore-test-results . Also It was passing as per the previous PR.

Reply:
Yes, I tried changing the configuration so that it starts working in this PR https://github.com/eugenp/tutorials/pull/4971/files, this Test is pretty old and I think it was working before, also PR #4971 got passed not sure how

Reply:
@dkapil That PR only updates the configuration in the maven project though and that's where the build passed. But the build of the full repo is still failing.

Reply:
@lor6 as per the full jenkins build here https://rest-security.ci.cloudbees.com/view/github%20projects/job/github-projects-Jobs/job/tutorials-build/103/ this maven test is failing the build. Let's merge this PR then.

Reply:
@dkapil  there's an issue with PRs from tutorials branches; I'll re-open this PR from my fork.